### PR TITLE
Updated Docs for Primary Validator - Accounts #962 Fix

### LINKED ISSUE #962: Update Docs for Primary Validator - Accounts
Update documentation and sample code for Primary Validator - Accounts
- ensure all available endpoints (`GET`, `POST`, `PATCH`, and `DELETE`) are documented
- update request and response examples from real API data (2 items for `GET` list responses)
- JSON should be formatted with 2 space indents
- ensure any/all params are documented in tables
- https://thenewboston.com/primary-validator-api/accounts

### PR DESCRIPTION
**Pull request by Hristijan95**
Fixes #962.
@buckyroberts, for the URL params (/accounts/<account_number>/balance for example) I think we are not adding them into tables, correct? If we are, I'll add a table, but as far as I could see, we only use tables for body params. If I am wrong, just let me know, I'll add the tables

### CODE REVIEW

**Comment by shahraizali** [comment]
Hi @Hristijan95! I confirmed with bucky and we did a small poll on the documentation channel as well we would be adding the query params to the tables as well. Look [here](https://github.com/thenewboston-developers/Website/pull/982/files#diff-c4eee41d3a8c9ab0d7ec0cb4717b0991755b71e565eb72fe46e6217c8b5be168) for reference

**Comment by Hristijan95** [comment]
Okay, great, I added the tables.

File: src/containers/PrimaryValidatorApi/PrimaryValidatorApiAccounts/index.tsx [hunk 1]
```diff
@@ -12,44 +12,55 @@ const PrimaryValidatorApiAccounts: FC = () => {
...
-    "balance_lock": "5e12967707909e62b2bb2036c209085a784fabbc3deccefee70052b6181c8ed8"
-  },
-  {
-    "id": "e8b24ad7-f81e-4731-818e-9bbbc3336428",
-    "account_number": "ad1f8845c6a1abb6011a2a434a079a087c460657aad54329a84b406dce8bf314",
-    "balance": "56.0000000000000000",
-    "balance_lock": "ad1f8845c6a1abb6011a2a434a079a087c460657aad54329a84b406dce8bf314"
-  }
-]`}
+        code={`{
```
Comment (shahraizali) [inline src/containers/PrimaryValidatorApi/PrimaryValidatorApiAccounts/index.tsx:15]: we are adding Table for Query params as well like **limit** and **offset**. kindly include that as well
Have a look here
https://github.com/thenewboston-developers/Website/pull/982/files#diff-c4eee41d3a8c9ab0d7ec0cb4717b0991755b71e565eb72fe46e6217c8b5be168

**Comment by Hristijan95** [comment]
@shahraizali I added the pagination query params, but I think we should make a difference between the query params & URL parameters, or I should remove the url/path parameters? 

To be more specific, I am talking about differentiating `/accounts/<account_number>/balance` and `/accounts?limit=50&offset=50`
Right now both are called "URL parameter". 
maybe account_number could stay "URL parameter" and the query params should be "Query parameter"? Let me know what you think.

**Comment by shahraizali** [comment]
How about renaming the one that is part of url to 'Path parameters' 
SOURCE: https://idratherbewriting.com/learnapidoc/docapis_doc_parameters.html

as far as the name URL parameter and Query Parameter I also looked it up a couple of days ago when I saw your PR. In my finding, they are used interchangeably and are arguably the same thing, In my opinion, that's fine.

**Comment by Hristijan95** [comment]
Agreed. I changed them to PATH.

**Review by shahraizali — APPROVED** [review]

**Pull request merged**